referents.md

```diff
@@ -39,7 +39,7 @@
 | ko | |
 | lt | |
 | lv | |
-| mk | |
+| mk | [@keljtanoski](https://github.com/keljtanoski) |
```

Comment:
Thank you for proposing as a referent, but we like to validate before

Reply:
@caouecs, Gotcha, sorry about this :)